sw/device/silicon_creator/lib/drivers/otbn.c

```diff
@@ -17,34 +17,16 @@
 #include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
 #include "otbn_regs.h"  // Generated.
 
-#define ASSERT_ERR_BIT_MATCH(enum_val, autogen_val) \
```

Comment:
@ballifatih this is an example for my comment [here](https://github.com/lowRISC/opentitan/pull/16802#discussion_r1047700310)